submissions/mustbefail/html-css-popup/index.html

```diff
@@ -1,10 +1,13 @@
 <!DOCTYPE html>
```

Comment:
Maybe you don't need to include these files from the popup task in this PR

Reply:
Could you tell me how to remove this?

Reply:
There a lot of techniques about how to do this.
But I think the simplest way to fix it is to replace these two files with the previous versions of them. You can copy their original versions from the Kottans repo [here](https://github.com/kottans/frontend-2021-homeworks/tree/main/submissions/mustbefail/html-css-popup). After doing this you can commit these changes and it will remove these changes from the current PR.

Reply:
Revert the state of these files to this commit [6d304c6](https://github.com/kottans/frontend-2021-homeworks/tree/main/submissions/mustbefail/html-css-popup), should fix the problem